Library/Backends/ZendEngine2/Backend.php

```diff
@@ -35,6 +35,9 @@ public function getVariableCodePointer(Variable $variable)
         return '&' . $variable->getName();
     }
 
+    /**
+     * {@inheritdoc}
```

Comment:
@sergeyklay 

It's notice that it can breaks some applications but for not it's working and most projects are using it
For example I am using inline plugin for phpstorm and atom for it


![image](https://cloud.githubusercontent.com/assets/572096/13265096/2d0bfd44-da82-11e5-8cc7-8aecd20ab53b.png)

![image](https://cloud.githubusercontent.com/assets/572096/13265001/cd575e48-da81-11e5-83d6-9e3ddc882a27.png)

https://github.com/symfony/symfony/search?utf8=%E2%9C%93&q=%7B%40inheritDoc%7D

![image](https://cloud.githubusercontent.com/assets/572096/13265034/ee2cb91a-da81-11e5-8baf-4a9c4b635382.png)

https://github.com/zendframework/zend-paginator/search?utf8=%E2%9C%93&q=%7B%40inheritDoc%7D

![image](https://cloud.githubusercontent.com/assets/572096/13265072/11c17118-da82-11e5-9f62-90b85dfbad6c.png)